tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/invocations/FuncInvocationExprTest.java

```diff
@@ -146,6 +147,23 @@ public Object[][] invocationWithArgVarargMixTestFunctions() {
         };
     }
 
+    @Test(expectedExceptions = {BLangRuntimeException.class},
+            expectedExceptionsMessageRegExp = "error: Invalid.*")
+    public void testFuncWithNeverReturnTypeWithoutVariableAssignment1() {
+        BRunUtil.invoke(funcInvocationExpResult, "testFuncWithNeverReturnTypeWithoutVariableAssignment1");
+    }
+
+    @Test(expectedExceptions = {BLangRuntimeException.class},
+            expectedExceptionsMessageRegExp = "error: Invalid.*")
+    public void testFuncWithNeverReturnTypeWithoutVariableAssignment2() {
+        BRunUtil.invoke(funcInvocationExpResult, "testFuncWithNeverReturnTypeWithoutVariableAssignment2");
+    }
```

Comment:
Panics can be tested in Ballerina itself. Trap in the test function and assert. You'll need an additional function in between.